src/components/List/__stories__/RowVariants.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import List from '..'
+import { MockData, generateData } from '../../../mocks/list'
+import { defaultColumns } from './helper'
+
+export const RowVariants = () => (
+  <List<MockData> columns={defaultColumns} data={generateData(3)} multiselect>
+    {list => (
+      <>
+        <list.Header />
+        <list.Body>
+          {({ rowData }) => (
+            <list.Row
+              alert={rowData.id === '1'}
+              locked={rowData.id === '2'}
+              animated
+              id={rowData.id}
+            >
+              <list.Cell>{rowData.name}</list.Cell>
+              <list.Cell>{rowData.description}</list.Cell>
+              <list.Cell>{rowData.department}</list.Cell>
+              <list.ExpendableContent>
+                {() => <>ExpendableContent of {rowData.name}</>}
+              </list.ExpendableContent>
+            </list.Row>
+          )}
+        </list.Body>
+      </>
+    )}
+  </List>
+)
+
+RowVariants.parameters = {
+  docs: {
+    storyDescription: `- Default
+- Alert
+- Locked`,
+  },
+}
```

Comment:
Can we spread props to avoid having this:

![Screenshot 2022-11-07 at 17 03 18](https://user-images.githubusercontent.com/15812968/200357177-ea5f2431-dc04-41a0-996a-46864804eb45.png)

Reply:
Storybook doesn't know how to parse the List content, unfortunately!